<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createvolume</class>
 <widget class="QDialog" name="createvolume">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>create a new volume</string>
  </property>
  <widget class="QLineEdit" name="lineEditVolumePath">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumePath">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>path to device</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCreate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>c&amp;reate</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;cancel</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>501</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbPassphrase">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPassphraseFromFile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key from a keyfile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassphrase1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOpenKeyFile">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>open a key file</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassPhrase">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassPhrase2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRepeatPassPhrase">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>repeat key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxFS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxVolumeType">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string notr="true">luks</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">plain</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxRNG">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">/dev/urandom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">/dev/random</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelvolumetype">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>volume type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>501</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditHiddenKey">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelHidden">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2Hidden">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>repeat key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHiddenKey1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHiddenKeyFile">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbHiddenKey">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>hidden key</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbHiddenKeyFile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>hidden key from keyfile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>volume size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHiddenSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxHiddenSize">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GB</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QLabel" name="labelrng">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>rng</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelfs">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>file system</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditVolumePath</tabstop>
  <tabstop>rbPassphrase</tabstop>
  <tabstop>rbPassphraseFromFile</tabstop>
  <tabstop>lineEditPassphrase1</tabstop>
  <tabstop>pbOpenKeyFile</tabstop>
  <tabstop>lineEditPassPhrase2</tabstop>
  <tabstop>comboBoxVolumeType</tabstop>
  <tabstop>rbHiddenKey</tabstop>
  <tabstop>rbHiddenKeyFile</tabstop>
  <tabstop>lineEditHiddenKey</tabstop>
  <tabstop>pbHiddenKeyFile</tabstop>
  <tabstop>lineEditHiddenKey1</tabstop>
  <tabstop>lineEditHiddenSize</tabstop>
  <tabstop>comboBoxHiddenSize</tabstop>
  <tabstop>comboBoxFS</tabstop>
  <tabstop>comboBoxRNG</tabstop>
  <tabstop>pbCreate</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
